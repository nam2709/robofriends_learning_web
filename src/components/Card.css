.light-pink-bg {
    background-color: #FFC0CB; /* Light Pink */
}