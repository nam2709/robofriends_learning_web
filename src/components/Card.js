import React from 'react';

const Card = ({ id, name }) => {
    return (
        <div className='tc bg-light-red dib br3 pa3 ma2 grow bw2 shadow-5'>
            {/* <img alt='robots' src={`https://robohash.org/${id}?size=200x200`}/> */}
            <img 
                alt='robots' 
                src={`https://shaded-adorable-doll.glitch.me/image/${id}`}
                style={{ width: "300px", height: "380px" }}
            />
            <div>
                <h2> {name} </h2>
                {/* <p> {email} </p> */}
            </div>
        </div>
    );
}

export default Card;