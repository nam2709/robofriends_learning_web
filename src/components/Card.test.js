import React from 'react';
import { render } from '@testing-library/react';
import Card from './Card';

// Ensure that Jest knows how to handle .snap files
import '@testing-library/jest-dom/extend-expect';

test('MyComponent snapshot', () => {
  const { asFragment } = render(<Card />);
  expect(asFragment()).toMatchSnapshot();
});