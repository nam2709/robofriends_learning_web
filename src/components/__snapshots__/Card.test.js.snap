// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent snapshot 1`] = `
<DocumentFragment>
  <div
    class="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/undefined?size=200x200"
    />
    <div>
      <h2>
          
      </h2>
      <p>
          
      </p>
    </div>
  </div>
</DocumentFragment>
`;
